source/guides/dashboard/runs.md

```diff
@@ -121,3 +121,17 @@ Runs that have been canceled or are in an errored state can be archived from the
   `https://dashboard.cypress.io/projects/{project ID}/runs/{run number}`
 - Click **{% fa fa-history %} Restore from archive**
   {% imgTag /img/dashboard/restore-from-archive.png "restore-from-archive" %}
+
+# Test Case History
+
+{% imgTag /img/dashboard/runs/dashboard-runs-details-sidebar-test-case-history.png "Screenshot of the Test Case History panel" %}
+
+When looking further into a test runs information, there is a new Test Case History panel. It matches test cases by spec and title and combines it with commit information to provide users insight into:
+
+1. See the history of a test over its lifespan
+1. View the diff of a single test case*
+1. Jump directly to the commit
+1. See what run number each test snapshot is associated with
```

Comment:
What's a 'test snapshot'? Is this an analytics term? Because this is a term already in the Test Runner for our 'time travel' feature, so I don't want this confused with that - as if we offer the viewing of these test snapshots in the Dashboard.

Reply:
That's not an analytics term, no. We don't really have much of a name for this concept. We can just remove this line from the docs – it's not a major selling point anyway.